src/app/app.component.spec.ts

```diff
@@ -19,6 +19,6 @@ describe('AppComponent', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.nativeElement as HTMLElement;
-    expect(compiled.querySelector('.content span')?.textContent).toContain('healthri-catalog-app app is running!');
+    expect(compiled.querySelector('.content span')?.textContent).toContain('gdi-catalogue-app app is running!');
```

Comment:
What do you think of renaming all references to `gdi-userportal`?

Reply:
I have changed it now to make it consistent. little bit confused. From my understanding the catalogue is part of the user portal. User portal will be a seperate project where all comes together?